README.md

```diff
@@ -1,9 +1,82 @@
-# React + Vite
+Books Management Application
+============================
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+Description
+-----------
 
-Currently, two official plugins are available:
+Cette application permet de gérer une liste de livres. Les utilisateurs peuvent filtrer les livres par sujet et niveau, sélectionner un livre pour voir ses détails et naviguer vers les chapitres spécifiques du livre.
```

Comment:
Using cascading PRs is convenient to avoid having similar changes to review on different PRs :)